.travis.yml

```diff
@@ -24,14 +15,14 @@ jobs:
     - stage: lint
       script:
         # Run linter
-        - npm run lint
+        - yarn lint
     - stage: test
       script:
         # Run tests
-        - npm run test
+        - yarn test
     - stage: build
       script:
         # Build all
-        - npm run build:docs
-        - npm run build:system
-        - npm run build:app
+        - yarn build:docs
+        - yarn build:system
+        - yarn build:app
```

Comment:
What was the reason again for switching over to yarn by default instead of npm?

Reply:
Since the cli-plugins must have a specific format for their name I was thinking of keeping vue-design-system a thing and create another package that would be named `vue-cli-plugin-vueds`.

Since it is cumbersome to have to maintain two repositories separately, mainly for testing, I was thinking that we could use yarn workspaces along with lerna to keep it all in here.

It works pretty well for me on [vue-cli-plugin-styleguidist](https://github.com/vue-styleguidist/vue-cli-plugin-styleguidist) and very well for Evan You on [@vue/cli](https://github.com/vuejs/vue-cli).

Do you understand where I am going with *yarn*?

Reply:
@elevatebart Makes sense to me!